python/torch_mlir/eager_mode/torch_mlir_tensor.py

```diff
@@ -0,0 +1,95 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+import warnings
+
+import torch
+from torch.utils._pytree import tree_map
+
+from torch_mlir.eager_mode.torch_mlir_dispatch import (
+    try_torch_mlir_eager,
+    UnsupportedByTorchMlirEagerMode,
+)
+from torch_mlir_e2e_test.linalg_on_tensors_backends import refbackend
+
+
+class TorchMLIRTensor(torch.Tensor):
```

Comment:
Is there any upstream documentation describing the extension point you are using here? (_make_wrapper_subclass/`__torch_dispatch_``, etc.) It would be good to link it in if it exists.

Reply:
I spoke with brian hirsch and he said the best documentation for how to use wrapper_subclass is https://github.com/albanD/subclass_zoo. I can add as a comment.